Cargo.toml

```diff
@@ -1,3 +1,25 @@
+[workspace]
+resolver = "2"
+members = [
+  "benches/benchsuite",
+  "benches/capture",
+  "crates/cargo-platform",
+  "crates/cargo-test-macro",
+  "crates/cargo-test-support",
+  "crates/cargo-util",
+  "crates/crates-io",
+  "crates/credential/*",
+  "crates/home",
+  "crates/mdman",
+]
+exclude = [
+  "crates/resolver-tests",
```

Comment:
I didn't see in the discussion, why is this excluded?

Reply:
`./x.py test tidy` fails for some licensing issue with resolver-tests:


```
tidy error: invalid license `BSD-2-Clause` in `cloudabi 0.0.3 (registry+https://github.com/rust-lang/crates.io-index)`
tidy error: dependency `fuchsia-cprng 0.1.1 (registry+https://github.com/rust-lang/crates.io-index)` does not define a license expression
tidy error: invalid license `0BSD` in `vec_mut_scan 0.3.0 (registry+https://github.com/rust-lang/crates.io-index)`
```

I can fix two of them but `fuchsia-cprng` is there without any license. Have no idea how to address that properly. I'll add a TODO above this line. Thanks for the reminder!

Reply:
Ah.  I posted #11886 to fix `fuchsia-cprng` and `cloudabi`.

I think the `vec_mut_scan` license exception is fine to add.

Reply:
Both PR in rust-lang/cargo and rust-lang/rust got updated. Thanks for the fix!